nemo_curator/download/commoncrawl.py

```diff
@@ -328,17 +359,29 @@ def extract_text(self, html, stop_words):
 
         if text is not None:
             paragraphs = list(filter(None, text.split("\n")))
-            result = []
-            for paragraph in paragraphs:
-                words = paragraph.split()
-                length = len(words)
-                if length == 0:
-                    continue
-                stopwords = [word for word in words if word in stop_words]
-                stopword_density = len(stopwords) / length
 
-                if stopword_density >= self.required_stopword_density:
-                    result.append(paragraph)
+            if language in NON_SPACED_LANGUAGES:
+                warnings.warn(
+                    "stopword_density is ignored for non-space-separated languages."
+                )
+                result = paragraphs
```

Comment:
Since https://github.com/NVIDIA/NeMo-Curator/pull/431 was just merged, I added this logic to the Trafilatura extractor. It is the same as the Resiliparse logic.